rollup-interface/src/state_machine/da.rs

```diff
@@ -19,17 +19,18 @@ pub trait DaSpec {
     /// A proof that each tx in a set of blob transactions is included in a given block.
     type InclusionMultiProof: Serialize + DeserializeOwned;
 
-    /// A proof that a claimed set of transactions is complete. For example, this could be a range
-    /// proof demonstrating that the provided BlobTransactions represent the entire contents of Celestia namespace
-    /// in a given block
+    /// A proof that a claimed set of transactions is complete.
+    /// For example, this could be a range proof demonstrating that
+    /// the provided BlobTransactions represent the entire contents
+    /// of Celestia namespace in a given block
     type CompletenessProof: Serialize + DeserializeOwned;
 
     /// The parameters of the rollup which are baked into the state-transition function.
     /// For example, this could include the namespace of the rollup on Celestia.
     type ChainParams;
 }
 
-/// A DaLayer implements the logic required to create a zk proof that some data
+/// A DaVerifier implements the logic required to create a zk proof that some data
```

Comment:
Again not sure about this but should it be:
```suggestion
/// A `DaVerifier` implements the logic required to create a zk proof that some data
```